cypress/support/e2e.js

```diff
@@ -17,4 +17,6 @@
 // import './commands'
 
 // Alternatively you can use CommonJS syntax:
-require('./commands');
\ No newline at end of file
+require('./commands');
+
+import '@testing-library/cypress/add-commands';
```

Comment:
I believe it's convention for this to go in `cypress/support/commands.js`, not `e2e.js`.